config.json

```diff
@@ -2,7 +2,8 @@
   "track_id": "gnu-apl",
   "language": "GNU APL",
   "active": false,
-  "test_pattern": "\\.tc$",
+  "test_pattern": "[.]tc$",
```

Comment:
This would be awesome as a glob instead of a regex, but I understand that that's not what the client expects.